readthedocs/builds/models.py

```diff
@@ -360,6 +358,10 @@ def get_github_url(
             return ''
         repo = repo.rstrip('/')
 
+        if not filename:
+            filename = ''
```

Comment:
We can receive filename as `None` or empty string, I can search for all the calls and make sure we always receive a string.

Reply:
I'm going to do that actually, we only use this in the footer api I think